internal/configs/version2/stream.go

```diff
@@ -15,9 +15,10 @@ type StreamUpstream struct {
 
 // StreamUpstreamServer defines a stream upstream server.
 type StreamUpstreamServer struct {
-	Address     string
-	MaxFails    int
-	FailTimeout string
+	Address        string
+	MaxFails       int
+	FailTimeout    string
+	MaxConnections int
```

Comment:
In general, we try to follow the names of NGINX directives/parameter in the config structs, because they represent NGINX configuration at NGINX level - so better if we have 1 to 1 match where possible for simplicity. Could you rename to `MaxConns` ?

Reply:
I think the name is fine as it is.